build.sbt

```diff
@@ -16,10 +16,31 @@ ThisBuild / tlSonatypeUseLegacyHost := false
 ThisBuild / tlSitePublishBranch := Some("main")
 
 val Scala213 = "2.13.8"
-ThisBuild / crossScalaVersions := Seq(Scala213, "3.1.2")
+val Scala3 = "3.1.2"
+
+ThisBuild / crossScalaVersions := Seq(Scala213, Scala3)
 ThisBuild / scalaVersion := Scala213 // the default Scala
 
-lazy val root = tlCrossRootProject.aggregate(core, java)
+ThisBuild / scalafixDependencies += "com.github.liancheng" %% "organize-imports" % "0.6.0"
+ThisBuild / semanticdbEnabled := !tlIsScala3.value
+ThisBuild / semanticdbVersion := scalafixSemanticdb.revision
+ThisBuild / scalafixScalaBinaryVersion := {
+  CrossVersion.binaryScalaVersion(scalaVersion.value)
+}
+
+ThisBuild / githubWorkflowBuild := {
+  val scalafixStep = WorkflowStep.Sbt(
+    List("root/scalafixAll --check"),
+    name = Some("Check that scalafix has been run"),
+    cond = Some(s"matrix.java == 'temurin@8' && matrix.scala != '$Scala3'")
+  )
+
+  scalafixStep +: (ThisBuild / githubWorkflowBuild).value
```

Comment:
Can be simplified after https://github.com/typelevel/sbt-typelevel/issues/306

Reply:
I thought about this a bit and have also been following the discussion in https://github.com/typelevel/otel4s/issues/3.

It seems like this project will likely have separate Scala 2 and Scala 3 sources. Therefore I propose the right thing to do here is to have two separate scalafix configurations, `.scalafix-2.conf` and `.scalafix-3.conf`. That way only the supported rules can be enabled in the `scalafix-3.conf`.

Reply:
`scalafixAll` requires an expensive compile step, so it should be done before the fast header/formatting checks.